owlcms/src/main/java/app/owlcms/monitors/EventForwarder.java

```diff
@@ -744,6 +760,8 @@ private void computeCurrentGroup(Group g) {
 
 		if (displayOrder != null && displayOrder.size() > 0) {
 			updateGroupInfo(computeLiftType(displayOrder.get(0)));
+			setLiftTypeKey(computeLiftTypeKey(displayOrder.get(0)));
+			setLiftType(computeLiftType(displayOrder.get(0)));

```

Comment:
I wasn't sure how important the pattern of getters and setters was for these fields. It feels like there should be a single method being called which will compute and set both values.

Reply:
Not terribly important.  Just following the general pattern.